test/geometry/GeoParamsTest.hh

```diff
@@ -0,0 +1,47 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file GeoParamsTest.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "gtest/Main.hh"
+#include "gtest/Test.hh"
+
+#include "geometry/GeoParams.hh"
+
+//---------------------------------------------------------------------------//
+// TEST HARNESS
+//---------------------------------------------------------------------------//
+
+namespace celeritas_test
+{
+using namespace celeritas;
+
+class GeoParamsTest : public celeritas::Test
+{
+  protected:
+    using SptrConstParams = std::shared_ptr<const GeoParams>;
+
+    static void SetUpTestCase()
+    {
+        std::string test_file
+            = celeritas::Test::test_data_path("geometry", "twoBoxes.gdml");
+        geom_ = std::make_shared<GeoParams>(test_file.c_str());
+    }
```

Comment:
Not blocking for this PR, but we should think about how this test harness could be changed if/when we want to try out multiple different GDML files inside a single `test.cc`. (This assumes VecGeom supports unloading a geometry and allocating a new, different one.)

Reply:
Agreed. I have created issue #43 for this purpose.